pages/productListingPage/productListingPage.css

```diff
@@ -0,0 +1,135 @@
+@import "/styles.css";
+
+:root {
+}
+
+body {
+}
+
+.nav {
+}
+
+.nav__logo {
+}
+
+.nav__search {
+}
+
+.nav__actions {
+}
+
+.nav__login-btn {
+    padding-left: 2.8rem;
+    padding-right: 2.8rem;
+}
+
+/* .nav__wish-lst-component {
+    position: relative;
+} */
+.nav__wish-lst-btn {
+    position: relative;
+    width: 2.5rem;
+    height: 2.2rem;
+    width: auto;
+    height: auto;
+    margin-left: 4rem;
+    margin-right: 4rem;
+}
+.nav__wish-lst-btn > div {
+    width: 2rem;
+    height: 2rem;
+}
+.nav__wish-lst-btn > svg {
+    width: 4rem;
+    height: 4rem;
+}
+
+.nav__card_lst-btn {
+    height: 2.3rem;
+    width: fit-content;
+    height: fit-content;
+    position: relative;
+    display: flex;
+    align-content: center;
+}
+.nav__card_lst-btn > div {
+    display: flex;
+    align-items: center;
+}
+.nav__card_lst-btn > div > svg {
+    width: 4rem;
+    height: 4rem;
+    margin-right: 1rem;
+}
+
+
+
+/* sidebar Starts */
+.sidebar {
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar__fliter {
+    display: flex;
+    margin-bottom: 2.4rem;
+}
+
+.sidebar__fliter button {
+    margin-left: auto;
+}
+
+.sidebar__price {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 2.4rem;
+}
+
+.sidebar__category {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 2.4rem;
+}
```

Comment:
price and category classes have the same properties. maybe make a generic class and re-use?

Reply:
Will make them generic when the same property is used a few more times.
These are initial screens and might change as the website evolves.

Also, nice suggestion for page-specific consolidated classes.